mopidy/core/actor.py

```diff
@@ -58,19 +58,27 @@ def __init__(self, mixer=None, backends=None):
         self.tracklist = TracklistController(core=self)
 
     def get_uri_schemes(self):
+        """Get list of URI schemes we can handle"""
         futures = [b.uri_schemes for b in self.backends]
         results = pykka.get_all(futures)
         uri_schemes = itertools.chain(*results)
         return sorted(uri_schemes)
 
     uri_schemes = property(get_uri_schemes)
```

Comment:
I don't care much for abstracting away the docstrings, but yes, I was thinking about using something like `deprecated_property()` to raise deprecation warnings as the next step.